physics/sfcsub.F

```diff
@@ -3645,14 +3744,14 @@ subroutine filanl(tsfanl,tsfan2,wetanl,snoanl,zoranl,albanl,      &
      &                  aisanl,                                         &
      &                  tg3anl,cvanl ,cvbanl,cvtanl,                    &
      &                  cnpanl,smcanl,stcanl,slianl,scvanl,veganl,      &
-     &                  vetanl,sotanl,alfanl,                           &
+     &                  vetanl,sotanl,socanl,alfanl,                    & !socanl: soil color
```

Comment:
same comment about only commenting on one variable -- it's a little strange and it's done several times. I appreciate the effort to explain, but maybe describe the variable once at the top of the routine?